kqueen_ui/blueprints/ui/test_views.py

```diff
@@ -170,3 +181,9 @@ def test_cluster_topology_data(client_login, cluster):
     response = client_login.get(url_for('ui.cluster_topology_data', cluster_id=cluster['id']))
     assert response.status_code == 200
     assert response.json == {}
+
+
+def test_set_cluster_network_policy(client_login, cluster):
```

Comment:
do u plan to create related tests, unset/failover?

Reply:
it is not really needed